### PR TITLE
Add literal extension

### PR DESCRIPTION
**Pull request by OkanBilal**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/literal/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "author": "OkanBilal",
+  "categories": [
+    "Web",
+    "Media"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "literal",
+      "title": "Search Books",
```
Comment (pernielsentikaer) [inline extensions/literal/package.json:16]:
```suggestion
      "title": "Search Books",
      "subtitle": "Literal",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Is there a way to move login to [preferences](https://developers.raycast.com/basics/prepare-an-extension-for-store#preferences) instead?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Automatically popping extension preferences is not something Raycast does, so I suggest removing that.

When you enter wrong credentials, I see two toasts shortly after each other. Could you look into only showing one on error?

**Comment by pernielsentikaer** [comment]
Is this ready for another review?

**Comment by OkanBilal** [comment]
Sure, you can review it now 👍

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
In the create review form you might want to use something like [useForm](https://developers.raycast.com/utilities/react-hooks/useform) for validating the input, it remembers states and handles validation pretty well 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**